shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -424,19 +492,21 @@ public Options getOptions() {
     outputFileOpt = new Option("o", "output", true, "local file to write the scan output to");
     namespaceOpt = new Option(ShellOptions.namespaceOption, "namespace", true,
         "namespace to display/set/delete properties for");
-
+    fileOpt = new Option("fs", "file", true, "file containing properties to set");
```

Comment:
I assume the `f` is used for filter?  `fs` (without context) seems like it should be for file system.  Would `p` and `propFile` work?

Reply:
I can definitely switch it to p for propFile.